Source/JavaScriptCore/inspector/agents/InspectorTargetAgent.cpp

```diff
@@ -166,7 +166,7 @@ void InspectorTargetAgent::didCommitProvisionalTarget(const String& oldTargetID,
 
 FrontendChannel::ConnectionType InspectorTargetAgent::connectionType() const
 {
-    return Ref { m_router }->hasLocalFrontend() ? Inspector::FrontendChannel::ConnectionType::Local : Inspector::FrontendChannel::ConnectionType::Remote;
+    return m_router->hasLocalFrontend() ? Inspector::FrontendChannel::ConnectionType::Local : Inspector::FrontendChannel::ConnectionType::Remote;
```

Comment:
do we not need a `CheckedRef { m_router }->` to use this properly?  i thought the counted value needs to be incremented in the caller scope

Reply:
A `const Ref` member is guaranteed to outlive a member function, so it does not need a local `Ref`.